dungeonGen.js

```diff
@@ -0,0 +1,284 @@
+$(function() {
+'use strict';
+
+	var sizeOpts, 
+		sizeChoice,
+		tSize,
+		points, 
+		posit, 
+		tile = $('canvas')[0].getContext('2d'), 
+		tileOut = $('canvas')[0].getContext('2d'),
+		width = $('canvas').width(), 
+		x = width/2,
+		height = $('canvas').height(), 
+		y = height/2,
+		coords = {"xPos":x, "yPos":y}; //default coordinates
+
+	//the initial black background
+	tile.fillStyle = "rgb(0,0,0)";
+	tile.fillRect(0, 0, width, height);
+
+	//disable the editing buttons upon app launch
+	$('#addButton').prop('disabled', true);
+	$('#undoButton').prop('disabled', true);
+
+	//Define a tile object
+	function Tile() {
+		this.inStack = [];
+		this.tileSize = tSize;
+
+		//find the index of the input coordinates in the tile array inStack
+		this.whereInStack = function(posit) {
```

Comment:
There are slightly more elegant ways of doing this kind of thing.

`var multArray = [[0,1], [5,5],[0,3], [0,2], [1,5]];
var posit = [0,3]

var result = multArray.filter(function(element, index, array) {
  console.log(element)
  return element[0] === posit[0] && element[1] === posit[1]
})

console.log(result[0], "found it")`